### ISSUE TITLE
Add term stocky habitus

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10408]
**Preferred term label:**
Stocky habitus

**Synonyms**
Stocky build

**Definition (free text, please give PubMed ID)**
Body type defined by short stature and overweight, muscular appearance. PMID: 6742769

**Parent term (use hpo.jax.org/app)**
Unsure about the best parent term. Our suggestion: Abnormality of body weight HP:0004323

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: Albright Hereditary Osteodystrophy PMID: 32644567
Hypochondroplasia (Bober MB, Bellus GA, Nikkel SM, et al. Hypochondroplasia. 1999 Jul 15 [Updated 2020 May 7]. In: Adam MP, Feldman J, Mirzaa GM, et al., editors. GeneReviews® [Internet]. Seattle (WA): University of Washington, Seattle; 1993-2024. Available from: https://www-ncbi-nlm-nih-gov.mu.idm.oclc.org/books/NBK1477/)
PMID: 3177454
PMID: 6742769

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
This is kind of a bundled term, not sure if it is good for HPO. Better to annotate short stature and overweight?
